enterprise/frontend/src/metabase-enterprise/remote_sync/components/GitSyncControls/GitSyncControls.unit.spec.tsx

```diff
@@ -247,4 +249,31 @@ describe("GitSyncControls", () => {
       });
     });
   });
+
+  describe("fullWidth prop", () => {
+    it("should render with default styling when fullWidth is false", async () => {
+      setup({ fullWidth: false });
+
+      await waitFor(() => {
+        expect(screen.getByTestId("git-sync-controls")).toBeInTheDocument();
+      });
+
+      const controls = screen.getByTestId("git-sync-controls");
+      expect(controls).toHaveStyle({ marginRight: "2rem" });
```

Comment:
Here and in the next test I think we should just check `width: 100%` - `.not.toHaveStyle({ width: "100%" });`, no need to check `margin`